flow/tutorials/README.md

```diff
@@ -0,0 +1,27 @@
+# OpenROAD-flow-scripts Tutorial
+
+The OpenROAD-flow-scripts tutorial to run the complete OpenROAD based
+flow from RTL-to-GDS using [OpenROAD Flow
+Scripts](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts).
+
+This tutorial also includes examples of useful design explorations and manual
+usage in key flow stages to help user gain a good understanding of the
+[OpenROAD](https://openroad.readthedocs.io/en/latest/main/README.html)
+application flow, data organization, GUI and commands. 
+
+Refer the flow tutorials documentation details 
+[here]((https://github.com/OpenROAD/blob/master/docs/tutorials/FlowTutorial.md)
```

Comment:
Let's wait for the OR PR and use the readthedocs link.

Reply:
@vijayank88 I agree Once the OR PR has been approved, make this final change

Reply:
@vijayank88 please update this with https://openroad.readthedocs.io/en/latest/tutorials/FlowTutorial.html